group_vars/Debian-stretch.yml

```diff
@@ -173,3 +173,4 @@ pkg_version_qtawesome: "0.7.2"
 pkg_version_pyvisa: "1.10.1"
 pkg_version_pyvisa_py: "0.3.1"
 pkg_version_ansible: "2.9.13"
+pkg_version_aiohttp: "3.7.4.post0"
```

Comment:
@anacso17 , we have been dropping subversioning within bugfixes, since one may work in a distribution and another one not. I suggest relaxing to "3.7.4*", if possible.

Reply:
done!